File: src/main/java/rs/raf/javaproject/model/BackupInfo.java
package rs.raf.javaproject.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalTime;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class BackupInfo {

    private String jobID;
    private String regionID;
    private List<Point> data;
    @JsonIgnore
    private LocalTime timestamp = LocalTime.now();

    @JsonIgnore
    public String getID() {
        return jobID + ":" + regionID;
    }
}

File: model/BackupInfo.py
class BackupInfo:

    def __init__(self):
        self.jobID = None
        self.regionID = None
        self.timestamp = None
        self.data = None

    def __init__(self, jobID, regionID, timestamp, data):
        self.jobID = jobID
        self.regionID = regionID
        self.timestamp = timestamp
        self.data = data
        
    def getID(self):
        return self.jobID + ":" + self.regionID
